tests/ml_models/test_mcorr_pytest.py

```diff
@@ -0,0 +1,412 @@
+# SPDX-License-Identifier: Apache-2.0
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+# Any modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+
+import torch
+
+from opensearch_py_ml.ml_models import MCorr
+
+
+def test_mcorr_output():
```

Comment:
Need to add more tests for this module. In addition to end-to-end performance, we probably want tests for individual components to localize errors or regressions.

Reply:
Agree